src/Microsoft.Health.Dicom.Core/Features/Workitem/WorkitemQueryTagStoreEntry.cs

```diff
@@ -24,5 +25,11 @@ public WorkitemQueryTagStoreEntry(int key, string path, string vr)
         /// Key of this extended query tag entry.
         /// </summary>
         public int Key { get; }
+
+        /// <summary>
+        /// Get the DicomItem that is the representation of the path for this tag
+        /// This is populated while fetching query tag from the db
+        /// </summary>
+        public DicomItem Item { get; set; }
```

Comment:
Is'nt DicomItem a Tag and VR, how can this represent the TagPath?

Reply:
Changed to list of dicomtags